moz_sql_parser/sql_parser.py

```diff
@@ -59,6 +59,8 @@ def nothing(*args):
     "between",
     "case",
     "collate nocase",
+    "columns",
```

Comment:
is `columns` keyword used?

Reply:
Yes Sir, We used this keyword for columns.
Example: `{'columns': [{'name': 'personid', 'type': 'int'}]`